Examples/Tests/plasma_lens/inputs_boosted_3d

```diff
@@ -0,0 +1,52 @@
+# Maximum number of time steps
+max_step = 84
+
+# number of grid points
+amr.n_cell =  16 16 16
+
+amr.max_level = 0
+
+# Geometry
+geometry.coord_sys   = 0                  # 0: Cartesian
+geometry.prob_lo     = -1.0  -1.0  -1.0   # physical domain
+geometry.prob_hi     =  1.0   1.0   2.0
+
+boundary.field_lo = pec pec pec
+boundary.field_hi = pec pec pec
+boundary.particle_lo = absorbing absorbing absorbing
+boundary.particle_hi = absorbing absorbing absorbing
+
+# Algorithms
+algo.particle_shape = 1
+warpx.gamma_boost = 2.
+warpx.boost_direction = z
```

Comment:
If this input file is identical to the one above (Examples/Tests/plasma_lens/inputs_3d), except for only these two boosted-frame parameters, we could also reuse the same input file and just set `runtime_params = warpx.gamma_boost=2 warpx.boost_direction=z` in WarpX-tests.ini. But I'm fine with keeping a separate input file, too. I think we haven't decided a policy when it comes to duplicating or not CI inputs.

Reply:
My preference is to have an input file for each set of inputs. Everything in the Examples directory is examples of inputs that users will see. I think it is better to have the examples in their full detail rather than hiding parameters in `WarpX-tests.ini`.